### PR TITLE
Update air conditioning example

### LINKED ISSUE #798: Restructure the examples in the documentation
Because @bernardokp keeps banging on about it.

I don't have any immediate plans to do this because I'd prefer people focused on the tutorials which are more fleshed out.

But PRs accepted from anyone interested.

The examples are https://github.com/odow/SDDP.jl/tree/master/docs/src/examples

### PR DESCRIPTION
**Pull request by Thuener**
Closes #798

- Add reference to Birge and Louveaux
- Show the optimal values and test them

### CODE REVIEW

**Comment by Thuener** [comment]
@odow how do you do format checks locally?

**Comment by odow** [comment]
https://github.com/odow/SDDP.jl/blob/b4c0a252916653e55f4f12c4beae5149f2a8f05f/.github/workflows/format_check.yml#L23-L27

**Comment by odow** [comment]
But I assumed @bernardokp wanted the examples fleshed out into proper literate tutorials (like the JuMP ones), not just some extra prints added.

**Comment by bernardokp** [comment]
Yeah, that would be ideal.

**Comment by Thuener** [comment]
Like this https://jump.dev/JuMP.jl/stable/tutorials/linear/diet/?

**Comment by bernardokp** [comment]
I mean, that would be a luxury!!!

**Comment by Thuener** [comment]
That will take some time

**Comment by odow** [comment]
> That will take some time

That's why I haven't done it :smile:

**Comment by Thuener** [comment]
I'm not crazy I asked before creating the PR 😅
image

**Comment by Thuener** [comment]
I can't promise anything now. @bernardokp you can help create the text? At least give me an initial version, I will extend it and create the code for it.

**Comment by bernardokp** [comment]
I certainly can help. As a matter of fact, it would be nice if we could create sort of a procedure to do it because I will probably create more examples in the very near future.

**Comment by bernardokp** [comment]
Is that what you want, @Thuener ?

The air conditioning example first appeared in the book Introduction to Stochastic Programming, by Birge and Louveaux, 1997, Springer-Verlag, New York.   It is on page 237, example 1. For a rescaled problem, they report an optimal first-stage solution of $x^1 = 2$, which corresponds to production in the model,  and $y^1 = 1$, which corresponds to stored_production.out in the model. Finally, they report an optimal value of 6.25.

File: docs/src/examples/air_conditioning.jl [hunk 1]
```diff
@@ -49,7 +55,19 @@ function air_conditioning_model(duality_handler)
...
+
+    sims = SDDP.simulate(model, 1, [:production, :stored_production])
+    x1 = sims[1][1][:production]
+    y1 = sims[1][1][:stored_production].out
+    @test isapprox(x1, 200, atol = 0.1)
+    @test isapprox(y1, 100, atol = 0.1)
+    println(
+        "With first stage soluctions $(x1) (production) and $(y1) (stored_production).",
+    )
+
```
Comment (odow) [inline docs/src/examples/air_conditioning.jl:70]:
```suggestion
```

File: docs/src/examples/air_conditioning.jl [hunk 2]
```diff
@@ -49,7 +55,19 @@ function air_conditioning_model(duality_handler)
         )
     end
     SDDP.train(model; duality_handler = duality_handler)
-    @test isapprox(SDDP.calculate_bound(model), 62_500.0, atol = 0.1)
+    lb = SDDP.calculate_bound(model)
+    println("Lower bound is: $lb")
+    @test isapprox(lb, 62_500.0, atol = 0.1)
+
```
Comment (odow) [inline docs/src/examples/air_conditioning.jl:61]:
```suggestion
```

**Pull request merged**